channels/facebook/multiple_post/index.js

```diff
@@ -65,21 +72,29 @@ function postMultipleMessages(params) {
  * @param index the array index of the message that needs to be sent
  * @returns {*}
  */
-function postMessage(sequenceName, params, responses, size, index) {
+function postMessage(sequenceName, params, responses, size, index, ow) {
   if (index < size) {
     const paramsForInvocation = Object.assign({}, params);
 
     if (Array.isArray(params.message)) {
       paramsForInvocation.message = params.message[index];
     }
-    return invokeAction(sequenceName, paramsForInvocation)
+
+    return invokeAction(sequenceName, paramsForInvocation, ow)
       .then(result => {
-        responses.push(result);
-        return postMessage(sequenceName, params, responses, size, index + 1);
+        responses.successfulPosts.push(result);
+        return postMessage(
+          sequenceName,
+          params,
+          responses,
+          size,
+          index + 1,
+          ow
+        );
       })
       .catch(e => {
-        responses.push(e);
-        return postMessage(sequenceName, params, responses, size, index + 1);
+        // Capture the response, don't send any further messages
```

Comment:
I have something in the works for a retry, I fear it will mean a lot of test updates so I was thinking of delivering it in another PR potentially after this one?

Reply:
Cool. That works. Just wanted to make sure we planned to do it. Thanks Dave!